src/main/kotlin/com/openlattice/graph/Graph.kt

```diff
@@ -487,6 +488,49 @@ class Graph(
         return neighbors
     }
 
+    override fun getEdgeEntitySetsConnectedToEntities(entitySetId: UUID, entityKeyIds: Set<UUID>): Set<UUID> {
+        val query = "SELECT DISTINCT ${EDGE_ENTITY_SET_ID.name} " +
+                "FROM ${E.name} " +
+                "WHERE (${SRC_ENTITY_SET_ID.name} = ? AND ${SRC_ENTITY_KEY_ID.name} = ANY(?) " +
```

Comment:
Just a minor thing, but can we use `SRC_IDS_SQL` and `DST_IDS_SQL` here?

Reply:
ooh true!